src/Core_Sideloader/UniversalAutoResolver/Core.UAR.StructReference.AI.cs

```diff
@@ -279,34 +279,59 @@ internal static class StructReference
 #endregion
 
 #region ChaFileClothes
-        private static Dictionary<CategoryProperty, StructValue<int>> ChaFileClothesGenerator()
+        private static Dictionary<CategoryProperty, StructValue<int>> ChaFileClothesGenerator(bool male, bool female)
```

Comment:
Can both be true or false at the same time?

Reply:
Yes, there's one spot where they do concatenated groupings of both sexes. I followed the same pattern as the character customization for that.